tests/mock_crust/churn.rs

```diff
@@ -213,6 +253,7 @@ impl ExpectedGets {
                 }
             }
         }
+        assert!(unexpected_receive <= self.sections.len());
```

Comment:
this is due to the scenario : 
1, A and B are tunnelling via C
2, A being the leader of a group message M (B is supposed to receive M)
3, A lost C (connection lost or C got removed), hence remove B
4, A collected enough signatures from section, hence sending M out according to its RT
   the sending list will contain an out-of-range node D
5, D also lost tunnel to a close-range node due to the removal of the tunnel_node
6, D will then receive that msg and handle it as from its perspective, it is within close-range.